tests/matrix_free/step-48.cc

```diff
@@ -330,6 +330,7 @@ namespace Step48
                                       norm_per_cell,
                                       QGauss<dim>(fe_degree + 1),
                                       VectorTools::L2_norm);
+    solution.zero_out_ghost_values();
```

Comment:
@kronbichler I am zeroing now the ghost values here.

Reply:
Would you have to do the same also in the step-48 tutorial program?

Reply:
Separately, though, it isn't clear to me why this is the right place for it. You don't use `solution` any more in the remainder of the function. From a readability perspective, I'd move the call to `zero_out_ghost_values()` to whatever the next place in control flow is that you need the ghost values to be zero.